src/adagrams.js

```diff
@@ -1,15 +1,102 @@
 export const drawLetters = () => {
-  // Implement this method for wave 1
-};
+  const LETTER_POOL = {
```

Comment:
Ohh, I kept it in all-caps since we treated it like a constant variable the first time. I wasn't confident about doing all-caps here instead of camelCase.